pulsar-broker/src/test/java/org/apache/pulsar/broker/service/PersistentTopicTest.java

```diff
@@ -663,6 +665,7 @@ private SubscriptionOption getSubscriptionOption(CommandSubscribe cmd) {
     @Test
     public void testSubscribeUnsubscribe() throws Exception {
         PersistentTopic topic = new PersistentTopic(successTopicName, ledgerMock, brokerService);
+        topic.initialize().join();
```

Comment:
Q: why add this?

Reply:
Because `createPersistentSubscription` depends on topicCompactionService object.